src/content/content.component.ts

```diff
@@ -17,97 +26,58 @@ import { BUTTON_TYPE } from '@shared-components/button/button.component';
 export class ContentComponent implements OnInit {
   value = '';
   SORT = SORT;
-  sortType: SORT = SORT.ASC;
   DISPLAY_TYPE = DISPLAY_TYPE;
   BUTTON_TYPE = BUTTON_TYPE;
   type: DISPLAY_TYPE = DISPLAY_TYPE.LIST;
   onlyFavoriteMovie = false;
+  allMovies: Movie[] = [];
   page: PageEvent = {
     pageIndex: DEFAULT_PAGE_INDEX,
     pageSize: DEFAULT_PAGE_SIZE,
     length: this.allMovies.length,
   };
   managedMovies: Movie[] = [];
 
-  constructor(public apiService: ApiService,
-              public moviesService: MoviesService,
-              public snackBar: MatSnackBar) {
-              }
-
+  constructor(public snackBar: MatSnackBar,
+              private readonly store: Store<AppState>) {
+  }
 
   ngOnInit(): void {
-    this.moviesService.setMoviesFromLocalStorage();
-    this.managedMovies = this.allMovies;
+    this.store.dispatch(downloadDataFromLocalStorage());
+    this.store.select(getAllMovies)
```

Comment:
czy potrzebujesz w komponencie `allMovies` i `managedMovies`? nie wystarczy zrobić `allMovies$ = this.store.select(getAllMovies)` i wtedy uzycie w HTMLu jako `allMovies$ | async`?